doc/cephfs/fs-volumes.rst

```diff
@@ -97,6 +97,19 @@ The CephX IDs authorized to <vol_name> need to be reauthorized to <new_vol_name>
 on-going operations of the clients using these IDs may be disrupted. Mirroring is
 expected to be disabled on the volume.
 
+Fetch the information of a CephFS volume using::
+
+    $ ceph fs volume info <vol_name>
+
+The output format is json and contains fields as follows.
```

Comment:
Can we add example of command output?
otherwise LGTM.

Reply:
I guess all the basic commands for volumes,subvolumegroups and subvolume doesn't have command output,its better to follow that:)